src/content/docs/infrastructure/amazon-integrations/get-started/introduction-aws-integrations.mdx

```diff
@@ -1,118 +1,48 @@
-    - /docs/accounts/install-new-relic/partner-based-installation/amazon-web-services-aws-users
-    - /docs/infrastructure/amazon-integrations/aws-integrations-list/aws-cloudformation-integration
-    - /docs/integrations/amazon-integrations/aws-integrations-list/aws-cloudformation-integration
-    - /docs/aws-cloudformation-integration
-    - /docs/infrastructure/amazon-integrations
-    - /docs/integrations/amazon-integrations
+  - /docs/integrations/amazon-integrations/get-started/introduction-aws-integrations
+  - /docs/infrastructure/amazon-integrations/getting-started/introduction-aws-integrations
+  - /docs/integrations/amazon-integrations/getting-started/introduction-aws-integrations
+  - /docs/accounts/install-new-relic/partner-based-installation/new-relic-aws-amazon-web-services
+  - /docs/partnerships/amazon-web-services-users
+  - /docs/accounts-partnerships/partnerships/partner-based-installation/amazon-web-services-aws-users
+  - /docs/accounts/install-new-relic/partner-based-installation/amazon-web-services-aws-users
+  - /docs/infrastructure/amazon-integrations/aws-integrations-list/aws-cloudformation-integration
+  - /docs/integrations/amazon-integrations/aws-integrations-list/aws-cloudformation-integration
+  - /docs/aws-cloudformation-integration
+  - /docs/infrastructure/amazon-integrations
+  - /docs/integrations/amazon-integrations
 freshnessValidatedDate: 2024-09-20
 ---
 
-New Relic offers several methods to collect performance data about your Amazon Web Services (AWS) environment. Once you've connected New Relic with AWS, you get total visibility into AWS telemetry data, letting you closely monitor the health of your AWS stack. 
+New Relic offers several methods to collect performance data about your Amazon Web Services (AWS) environment. Once you've connected New Relic with AWS, you can access your system's AWS telemetry, letting you closely monitor the health of your AWS stack. This doc describes how to get started with AWS monitoring and provides an overview about AWS costs. 
 
-## Integrate New Relic with your AWS stack [#connect]
+## Start with the CloudWatch Metric Streams integration [#monitor-aws]
 
-To get started, you need to establish a connection between New Relic and your AWS services. Establishing the initial connection integrates New Relic into your AWS stack so you can access additional integrations within the AWS ecosystem. 
+Our AWS observability solution is the Amazon CloudWatch Metric Streams integration, which collects data about your entire AWS stack with [Amazon CloudWatch Metric Streams](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-Metric-Streams.html). For the majority of use cases, you begin monitoring your AWS stack by [installing our Amazon CloudWatch Metric Streams integration](/docs/infrastructure/amazon-integrations/connect/cloudwatch-metric-streams/aws-metric-stream-setup).
```

Comment:
@akristen I think the link to the main installation page is now broken: [installing our Amazon CloudWatch Metric Streams integration](/docs/infrastructure/amazon-integrations/connect/cloudwatch-metric-streams/aws-metric-stream-setup).

Reply:
Added the redirect and updated the url